tools/RAiDER/cli/raiderDelay.py

```diff
@@ -190,11 +193,6 @@ def read_inps2run_steps(inps, step_list):
 
         run_steps = step_list[idx0:idx1]
 
-    # empty the step list if:
```

Comment:
Does this work ok? I.e. if you pass raiderDelay.py -g you should print the config then exit.

Reply:
Yes, this is redundant. we already check for this on lines 134